spec/text-objects-spec.coffee

```diff
@@ -205,15 +205,36 @@ describe "TextObjects", ->
       expect(editorElement.classList.contains('normal-mode')).toBe(true)
 
     it "selects inside the current paragraph in visual mode", ->
+
```

Comment:
this seems to be an unnecessary empty line; with or without it I'm still recommending an immediate merge for this PR 8-)